connector_importer/utils/mapper_utils.py

```diff
@@ -0,0 +1,306 @@
+# Author: Simone Orsi
+# Copyright 2018 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from datetime import datetime
+
+import pytz
+
+from odoo import fields
+
+from ..log import logger
+
+FMTS = ("%d/%m/%Y",)
+
+FMTS_DT = ("%Y-%m-%d %H:%M:%S", "%Y-%m-%d %H:%M:%S.000")
+
+
+def to_date(value, formats=FMTS):
```

Comment:
@guewen time to move some of these utils to connector or to a side module?

Reply:
Moving them to connector makes sense.

Reply:
@sebalix please add this to the roadmap and address it while working on our importers. Ok?

Reply:
@simahawk done